doc/other_stuff.md

```diff
@@ -0,0 +1,162 @@
+# Other important stuff
```

Comment:
Not sure if reviewing the docs should also be part of this PR but I think we could try to get rid of this whole file and move its content to the other sections.